nextflow_schema.json

```
@@ -327,7 +327,7 @@
                 },
                 "cf_ploidy": {
                     "type": "string",
-                    "default": 2,
+                    "default": "2",
```

what happened here?

---

hu must have happened when I ran `nf-core pipelines schema build`

---

can you revert this? this causes issues otherwise. 
```suggestion
                    "default": 2,
```